privi.Procfile

```diff
@@ -1,3 +1,3 @@
-privi: air --build.cmd "go build -o bin/privi cmd/privi/main.go" --build.bin "bin/privi" 
-funnel-privi: go run cmd/funnel/main.go 8080 privi 
+privi: air --build.cmd "go build -o bin/privi ./cmd/privi" --build.bin "bin/privi" -- --profile cmd/privi/dev.yaml --port 8080
+funnel-privi: go build -o ./bin/funnel ./cmd/funnel; ./bin/funnel 8080 privi
```

Comment:
can we separate into `build-privi` and `run-privi` commands? and then `run-privi` can take in `--funnel` arg?

i feel like it's pretty often i wanna re-build and not re-run and vice versa

Reply:
No this procfile is exclusively for running. During development you do gmake privi-dev and it will start these 4 processes in parallel. The makefile should contain all the build commands.